### ISSUE TITLE
Gradle needs updating to kotlin

### ISSUE THREAD

**Issue by scottbiggs** [android/codelab-android-room-with-a-view#263]
The gradle code samples in page 4 (https://developer.android.com/codelabs/android-room-with-a-view-kotlin#3) are all in groovy, which is being replaced by kotlin (didn't you know that?).

Please add a kotlin gradle version of this code.  Converting between the old groovy and the new kotlin format is extremely tricky for beginners (I gave up).

**Comment by cyCaffine** [comment]
Hi, Please assign this issue to me, I can solve this.

**Comment by Nick-P-M** [comment]
When you do this please also migrate the sample code from kapt to ksp.

**Comment by HackJJ** [comment]
Agreed, it's pretty hard to follow the code when the version of Android Studio you have (latest version) isn't comptiable. Makes it hugely frustrating.

**Comment by dhsilber** [comment]
> Agreed, it's pretty hard to follow the code when the version of Android Studio you have (latest version) isn't comptiable. Makes it hugely frustrating.

I've just run into this issue in attempting to work through this codelab. If/when someone updates this, I would like to give it a test run.
